content/influxdb/clustered/install/configure-cluster/use-helm.md

```diff
@@ -0,0 +1,964 @@
+```
+
+By default, this secret is named `gar-docker-secret`.
+If you change the name of this secret, you must also change the value of the
+`imagePullSecrets.name` field in your `values.yaml`.
+
+<!---------------------------- END Public Registry ---------------------------->
+
+{{% /tab-content %}}
+{{% tab-content %}}
+
+<!--------------------------- BEGIN Private Registry -------------------------->
+
+#### Private registry (air-gapped)
+
+If your Kubernetes cluster can't use a public network to download container images
+from our container registry, do the following:
+
+1.  Copy the images from the InfluxDB registry to your own private registry.
+2.  Configure your `AppInstance` resource with a reference to your private
+    registry name.
+3.  Provide credentials to your private registry.
+
+The list of images that you need to copy is included in the package metadata.
+You can obtain it with any standard OCI image inspection tool. For example:
+
+{{% code-placeholders "PACKAGE_VERSION" %}}
+
+```sh
+DOCKER_CONFIG=/tmp/influxdbsecret \
+crane config \
+  us-docker.pkg.dev/influxdb2-artifacts/clustered/influxdb:PACKAGE_VERSION \
+  | jq -r '.metadata["oci.image.list"].images[]' \
+  > /tmp/images.txt
+```
+
+{{% /code-placeholders %}}
+
+The output is a list of image names, similar to the following:
+
+```
+us-docker.pkg.dev/influxdb2-artifacts/idpe/idpe-cd-ioxauth@sha256:5f015a7f28a816df706b66d59cb9d6f087d24614f485610619f0e3a808a73864
+us-docker.pkg.dev/influxdb2-artifacts/iox/iox@sha256:b59d80add235f29b806badf7410239a3176bc77cf2dc335a1b07ab68615b870c
+...
+```
+
+Use `crane` to copy the images to your private registry:
+
+{{% code-placeholders "REGISTRY_HOSTNAME" %}}
+
+```sh
+</tmp/images.txt xargs -I% crane cp % REGISTRY_HOSTNAME/%
+```
+
+{{% /code-placeholders %}}
+
+---
+
+Replace {{% code-placeholder-key %}}`REGISTRY_HOSTNAME`{{% /code-placeholder-key %}}
+with the hostname of your private registry--for example:
+
+```text
+myregistry.mydomain.io
+```
+
+---
+
+Set the
+`images.registryOverride` field in your `values.yaml` to the location of your
+private registry--for example:
+
+{{% code-placeholders "REGISTRY_HOSTNAME" %}}
+
+```yml
+images:
+  registryOverride: REGISTRY_HOSTNAME
+```
+
+{{% /code-placeholders %}}
+
+<!---------------------------- END Private Registry --------------------------->
+
+{{% /tab-content %}}
+{{< /tabs-wrapper >}}
+
+### Set up cluster ingress
+
+{{% note %}}
+InfluxDB Clustered components use gRPC/HTTP2 protocols. If using an external load balancer,
+you may need to explicitly enable these protocols on your load balancers.
+{{% /note %}}
+
+The InfluxDB Clustered Helm chart includes the
+[Kubernetes Nginx Ingress Controller](https://github.com/kubernetes/ingress-nginx).
+Add a valid TLS Certificate to the cluster as a secret.
+Provide the paths to the TLS certificate file and key file:
+
+{{% code-placeholders "TLS_(CERT|KEY)_PATH" %}}
+
+```sh
+kubectl create secret tls ingress-tls \
```

Comment:
Should users still use `kubectl` to manage secrets or can/should this be done with Helm?

Reply:
My understanding is that secret management is not (and should not) be managed with Helm. Helm 3 will use the k8s default storage driver for secrets. 

@jenniferplusplus Please correct me if this doesn't sound right!

Reply:
That's correct. I would recommend people to still use kubectl to create the handful of secrets we need. If there's a way to do that with helm, I've never encountered it.